templates/field/field--text-long.html.twig

```diff
@@ -6,18 +6,16 @@
  * @see ./core/themes/stable/templates/field/field.html.twig
  */
 #}
-{% apply spaceless %}
-  {% if not label_hidden %}
-    {%
-      set title_classes = [
-      'ecl-u-type-bold',
-      'ecl-u-mb-m',
-      label_display == 'visually_hidden' ? 'visually-hidden',
-    ]
-    %}
-    <div{{ title_attributes.addClass(title_classes) }}>{{ label }}</div>
-  {% endif %}
-  <div class="ecl">
-    {%- include 'field--bare.html.twig' -%}
-  </div>
-{% endapply %}
+{% if not label_hidden %}
```

Comment:
```suggestion
{%- if not label_hidden -%}
```
Instead of the spaceless we can use the "-" in this and more places.
Not sure how many we need.
The purpose, from what I understand, is to remove whitespace caused by the template itself.

Reply:
Good point, thanks!